driver/nbd_protocol.c

```diff
@@ -139,16 +140,17 @@ NbdReadHandshakeReply(_In_ INT Fd)
     Retval->Datasize = RtlUlongByteSwap(Retval->Datasize);
 
     if (REPLY_MAGIC != Retval->Magic) {
-        WNBD_LOG_ERROR("Received invalid negotiation magic %llu (expected %llu)",
-                       Retval->Magic, REPLY_MAGIC);
+        WNBD_LOG_WARN("Received invalid negotiation magic %llu (expected %llu)",
```

Comment:
I'm wondering if we should stick to WNBD_LOG_ERROR. It's a fatal error, meaning that the NBD connection cannot be established.